### PR TITLE
Arrange winter.pagination package across web, drf and schema

### PR DESCRIPTION
**Pull request by mofr**

### CODE REVIEW

File: tests/pagination/test_page_position_argument_inspector.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 import winter
 from winter.data.pagination import PagePosition
-from winter.pagination import PagePositionArgumentResolver
-from winter.pagination import PagePositionArgumentsInspector
+from winter.web.pagination import PagePositionArgumentResolver
```
Comment (pristupa) [inline tests/pagination/test_page_position_argument_inspector.py:6]: not sorted ;-)

File: winter/drf/page_serializer.py [hunk 2]
```diff
@@ -5,9 +5,9 @@
 from rest_framework.request import Request as DRFRequest
 from rest_framework.serializers import SerializerMetaclass
 
-from .utils import get_next_page_url
-from .utils import get_previous_page_url
-from ..data.pagination import Page
+from winter.web.pagination.utils import get_next_page_url
+from winter.web.pagination.utils import get_previous_page_url
```
Comment (pristupa) [inline winter/drf/page_serializer.py:9]: not sorted ;-)

File: winter/schema/__init__.py [hunk 3]
```diff
@@ -4,11 +4,18 @@
 from .method_arguments_inspector import MethodArgumentsInspector
 from .method_arguments_inspector import get_method_arguments_inspectors
 from .method_arguments_inspector import register_controller_method_inspector
+from .page_position_argument_inspector import PagePositionArgumentsInspector
```
Comment (pristupa) [inline winter/schema/__init__.py:7]: not sorted ;-)

File: winter/web/pagination/order_by.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
 from .check_sort import check_sort
 from .order_by_annotation import OrderByAnnotation
 from .parse_sort import parse_sort
-from ...core.annotation_decorator import annotate_method
+from winter.core.annotation_decorator import annotate_method
```
Comment (pristupa) [inline winter/web/pagination/order_by.py:6]: I like when absolute imports is a separate group of imports, maybe joined to a group of 3rd party libraries:

```
import enum

from django.conf import settings
from winter.core.annotation_decorator import annotate_method

from .check_sort import check_sort
from .order_by_annotation import OrderByAnnotation
from .parse_sort import parse_sort
```

File: winter/schema/__init__.py (hunk 3 above)
Comment (mofr) [reply winter/schema/__init__.py:7]: Man, it looks sorted here.

File: winter/web/pagination/order_by.py (hunk 4 above)
Comment (mofr) [reply winter/web/pagination/order_by.py:6]: It's formatted so by IDE because it's from the same library.

**Pull request merged**